css/src/components/form/textarea.scss

```diff
@@ -145,4 +148,14 @@ $textarea-divider-radius: 0.5rem !default;
 		gap: $textarea-form-gap;
 		height: 100%;
 	}
+
+	.textarea-form-help {
+		display: flex;
+		margin-inline-start: $textarea-margin-spacing;
+
+		.help {
+			align-items: center;
```

Comment:
Did you mean to center the help message vertically against the icon? 

![image](https://github.com/user-attachments/assets/0faf98bb-09c0-4d06-8f40-4d7067ebaa27)

If so, you'll also need to add `display: flex` in order for it to center

![image](https://github.com/user-attachments/assets/5e411165-7ae2-46fe-8eec-eb652dc96f68)

Reply:
oh yes, updated